conjur/wrapper/http_wrapper.py

```diff
@@ -68,8 +74,8 @@ def invoke_endpoint(http_verb, endpoint, params, *args, check_errors=True,
             response.raise_for_status()
         except requests.exceptions.HTTPError as http_error:
             if response.text:
-                logging.debug(requests.exceptions.HTTPError(f'{http_error.response.status_code}' \
-                                                            f' {http_error.response.reason}' \
+                logging.debug(requests.exceptions.HTTPError(f'{http_error.response.status_code}'
```

Comment:
Just noticed that in some of your logs you do '' and in some you do ""

Lets try to be consistent with this

Reply:
updated this spot. The rest are in testing files I believe so I prefer not to touch them in this PR because it will get messy and cloud this PR